projects/BFDR/IJENatality.cs

```diff
@@ -4592,12 +4606,18 @@ public string ATTEND_OTH_TXT
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: 
+                if (record.AttendantOtherHelper != null)
+                {
+                    return record.AttendantOtherHelper.Trim();
```

Comment:
Since this is a text field, it should use a LeftJustified helper to add the additional whitespace for this IJE field so it's the correct length.